HyperV/scripts/EnsureOpenStackServices.ps1

```diff
@@ -407,11 +407,24 @@ Function Check-Service
     $filter='Name=' + "'" + $serviceName + "'"
 
     #Temporary hack
+	
+    Stop-Service $serviceName
+	
     $service=Get-WmiObject -namespace "root\cimv2" -Class Win32_Service -Filter $filter
     if($service)
     {
         $service.delete()
     }
+    #to make sure the service was actually deleted
+    Get-Service $serviceName 2> $null
```

Comment:
Use powershell's specific error handling (i.e -ErrorAction SilentlyContinue) instead of the linux-style redirect.